pkg/cli/environment.go

```diff
@@ -87,8 +86,6 @@ type EnvSettings struct {
 func New() *EnvSettings {
 	// TODO check with reflection that we are not missing any field
 	env := &EnvSettings{
-		EnvSettings: nil, // this is nil it gets overwritten below to ensure that it does not touch Helm's stuff
```

Comment:
If you remove this, this will definitely break things because then the `New()` function of helms cli will definitely be run.

Reply:
do we care? I dont think this is necessary any longer as we are already doing `env.EnvSettings = cli.New()` on line 114 and were already. This line wasn't needed before I think

Reply:
If I remember correctly there was some weird behaviour with composition, that's why we explicitly set that.

Reply:
tested it locally, and I think this is line is not needed, indeed.

Reply:
then go ahead :)